### PR TITLE
feat: [PL-27393]: Revert Switch account changes

### PR DESCRIPTION
**Pull request by avinashmadhwani02**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10529)

File: src/services/portal/overrides.yaml [hunk 1]
```diff
@@ -104,4 +103,3 @@ operationIdOverrides:
   /delegates-verification/initialized: verifyDelegateInitialization
   /delegates-verification/initializedV2: verifyDelegateInitializationV2
   /awshelper/aws-regions: getAwsRegions
-  /users/userAccounts: getUserAccounts
```
Comment (abhinavrastogi-harness) [inline src/services/portal/overrides.yaml:107]: no change in portal service?

**Review by pbarapatre10 — APPROVED** [review]
LGTM!

File: src/services/portal/overrides.yaml (hunk 1 above)
Comment (avinashmadhwani02) [reply src/services/portal/overrides.yaml:107]: There were many conflicts and types were breaking, so kept it as it is

**Review by abhinavrastogi-harness — DISMISSED** [review]

**Review by pbarapatre10 — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**